src/nvim/ex_cmds2.c

```diff
@@ -57,23 +57,23 @@ typedef struct scriptitem_S {
   char_u      *sn_name;
   bool file_id_valid;
   FileID file_id;
-  int sn_prof_on;               /* TRUE when script is/was profiled */
-  int sn_pr_force;              /* forceit: profile functions in this script */
-  proftime_T sn_pr_child;       /* time set when going into first child */
-  int sn_pr_nest;               /* nesting for sn_pr_child */
-  /* profiling the script as a whole */
-  int sn_pr_count;              /* nr of times sourced */
-  proftime_T sn_pr_total;       /* time spent in script + children */
-  proftime_T sn_pr_self;        /* time spent in script itself */
-  proftime_T sn_pr_start;       /* time at script start */
-  proftime_T sn_pr_children;    /* time in children after script start */
-  /* profiling the script per line */
-  garray_T sn_prl_ga;           /* things stored for every line */
-  proftime_T sn_prl_start;      /* start time for current line */
-  proftime_T sn_prl_children;    /* time spent in children for this line */
-  proftime_T sn_prl_wait;       /* wait start time for current line */
-  int sn_prl_idx;               /* index of line being timed; -1 if none */
-  int sn_prl_execed;            /* line being timed was executed */
+  int sn_prof_on;               // TRUE when script is/was profiled
+  int sn_pr_force;              // forceit: profile functions in this script
+  proftime_T sn_pr_child;       // time set when going into first child
+  int sn_pr_nest;               // nesting for sn_pr_child
+  // profiling the script as a whole
+  int sn_pr_count;              // nr of times sourced
+  proftime_T sn_pr_total;       // time spent in script + children
+  proftime_T sn_pr_self;        // time spent in script itself
+  proftime_T sn_pr_start;       // time at script start
+  proftime_T sn_pr_children;    // time in children after script start
+  // profiling the script per line
+  garray_T sn_prl_ga;           // things stored for every line
+  proftime_T sn_prl_start;      // start time for current line
+  proftime_T sn_prl_children;    // time spent in children for this line
+  proftime_T sn_prl_wait;       // wait start time for current line
+  long sn_prl_idx;               // index of line being timed; -1 if none
```

Comment:
@cacplate Should I cast things to int instead? Since this is index of a line, probably it should be linenr_T?